app/src/index.js

```diff
@@ -2,11 +2,24 @@
 /// user with a visual representation of this process (welcome screen). It also implements a view
 /// allowing to choose a debug rendering test from.
 
+var is_electron = window && window.process && window.process.type;
+var static_path;
+
+if (is_electron) {
+    // To avoid compile-time evaluation
+    eval("window.NODE_ENV=process.env.NODE_ENV; window.DIR_NAME = __dirname");
+    if (NODE_ENV == "development") {
+        static_path = "./"
+    } else {
+        static_path = DIR_NAME.replace(/app\.asar$/, 'static')+"/";
+    }
```

Comment:
How does it work? What happens when you replace `app.asar` with `static` and why we are doing it?